buildenv/jenkins/common/pipeline-functions

```diff
@@ -152,6 +152,9 @@ def workflow() {
                 case "_extended":
                     TEST_JOB_NAME = "Test-Extended-JDK${SDK_VERSION}-${SPEC}"
                     break
+                case "_experimental":
+                    TEST_JOB_NAME = "Test-Experimental-JDK${SDK_VERSION}-${SPEC}"
+                    break
```

Comment:
Don't need this for the PR build, only if you plan on adding a standalone test job.